shared/django_apps/dummy_settings.py

```diff
@@ -28,6 +28,7 @@
     "shared.django_apps.codecov_auth",
     "shared.django_apps.core",
     "shared.django_apps.reports",
+    "shared.django_apps.bundle_analysis_app",
```

Comment:
Any reason why we don't call this "bundle_analysis"?

Reply:
Was getting an error in API repo when using "bundle_analysis", think Django is getting it confused with some other directory in the repo.